scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -147,14 +147,14 @@ declare namespace WebdriverIO {
     interface Hooks extends _HooksArray, _Hooks { }
 
     type ActionTypes = 'press' | 'longPress' | 'tap' | 'moveTo' | 'wait' | 'release';
-    interface TouchAction {
+    interface TouchAction<T> {
         action: ActionTypes,
         x?: number,
         y?: number,
-        element?: Element,
+        element?: T,
```

Comment:
Can a touch action be done in something else that is not an Element?

Reply:
yes, the property is optional so you can do an action on arbitrary coordinates

Reply:
@SanthoshBonala it seems that @baruchvlz is correct and this property only takes Element types and nothing else